Userland/Libraries/LibCore/Version.h

```diff
@@ -12,4 +12,6 @@ namespace Core::Version {
 
 constexpr StringView SERENITY_VERSION = "Version 1.0"sv;
 
+String long_version_string();
```

Comment:
Maybe it would be better if we explicitly say that the function reads version from file, something like `read_long_version_string`?